package/kedro_viz/integrations/kedro/hooks_utils.py

```diff
@@ -166,3 +166,27 @@ def is_default_run(run_params: dict) -> bool:
 
     # True if none of the filtering params are set (i.e. full/default run)
     return not any(filtering_params)
+
+
+def is_sequential_runner(run_params: dict) -> bool:
+    """
+    Check if the run is using SequentialRunner.
+
+    Args:
+        run_params: Dictionary containing pipeline run parameters
+
+    Returns:
+        bool: True if the run is using SequentialRunner, False otherwise
+    """
+    runner = run_params.get("runner")
+
+    # If no runner specified, default is SequentialRunner
+    if runner is None:
+        return True
+
+    # If runner is a string, check if it contains "SequentialRunner"
+    if isinstance(runner, str):
+        return "SequentialRunner" in runner
```

Comment:
This logic is a bit weird 😅 what if I have a custom runner called `AwesomeSequentialRunner` or even `my_library.SequentialRunner` that happens to be incompatible with Kedro-Viz in some way? We have accepted that Kedro Viz has issues with the `ParallelRunner` but I think this is a bit of overshooting. @merelcht @ElenaKhaustova what do you think?